addons/proton_scatter/src/scatter.gd

```diff
@@ -257,7 +266,7 @@ func _rebuild(force_discover) -> void:
 func _rebuild_threaded() -> void:
 	# Disable thread safety, but only after 4.1 beta 3
 	if _thread.has_method("set_thread_safety_checks_enabled"):
-		_thread.set_thread_safety_checks_enabled(false)
+		Thread.set_thread_safety_checks_enabled(false)
```

Comment:
4.0.2 doesn't like this.

```
SCRIPT ERROR: Parse Error: Static function "set_thread_safety_checks_enabled()" not found in base "Thread".
          at: GDScript::reload (res://addons/proton_scatter/src/scatter.gd:263)
SCRIPT ERROR: Invalid call. Nonexistent function 'is_thread_running' in base 'Node3D ()'.
          at: _redraw (res://addons/proton_scatter/src/scatter_gizmo_plugin.gd:42)
SCRIPT ERROR: Invalid set index 'undo_redo' (on base: 'Node3D ()') with value of type 'EditorUndoRedoManager'.
          at: _on_selection_changed (res://addons/proton_scatter/plugin.gd:102)
SCRIPT ERROR: Invalid call. Nonexistent function 'is_thread_running' in base 'Node3D ()'.
          at: _redraw (res://addons/proton_scatter/src/scatter_gizmo_plugin.gd:42)
```

Opening up scatter.gd it has 263 highlighted.

Reverting it to `_thread` is fine.

Reply:
So that throws on the parser side, even before it even runs. I'll revert it and live with the warning then.